server/plugins/connect-plugin-api/src/main/java/com/oceanbase/odc/plugin/connect/api/SessionExtensionPoint.java

```diff
@@ -34,4 +34,6 @@ public interface SessionExtensionPoint extends ExtensionPoint, SessionOperations
     String getCurrentSchema(Connection connection);
 
     String getVariable(Connection connection, String variableName);
+
+    String getAlterSessionVariableStatement(String variableScope, String variableName, String variableValue);
```

Comment:
the method name is not so good, we should not emphasize conference in method names